client/Dockerfile.fedora

```diff
@@ -1,11 +1,10 @@
-FROM fedora:25
+FROM fedora:26
 MAINTAINER Luiz Carvalho <lucarval@redhat.com>
 
 ARG UPDATES_TESTING
 
 RUN dnf update -y && dnf install -y dnf-plugins-core && \
-    if [ $UPDATES_TESTING ]; then dnf config-manager --set-enable updates-testing; fi && \
-    dnf -y update && \
+    if [ $UPDATES_TESTING ]; then dnf config-manager --set-enable updates-testing && dnf -y update; fi && \
```

Comment:
We should really remove any sort of dnf/yum update from all these Dockerfiles. If the concern is stale packages, we should just update the parent image.